src/Acme.UI/obj/Debug/Acme.UI.csproj.CoreCompileInputs.cache

```diff
@@ -1 +1 @@
-e3affabcd405137f2bd4a95492b9fd1491556aa1
+f3621f8575353c71ac029cdbade5297303d22d1e
```

Comment:
I dont think this needs to be source controlled, I would suggest exlcluding all of /obj/